navpy/utils/utils.py

```diff
@@ -54,6 +54,32 @@ def input_check_Nx3(x):
     return x, N
 
 
+def input_check_Nx3x3(x):
+    """
+    Check x to be of dimension Nx3x3
+
+    Adhika Lie
```

Comment:
Feel free to put your name or remove.  I think this was left over from copy past of Adhika's input check code.